python/mxnet/test_utils.py

```diff
@@ -1933,7 +1933,7 @@ def chi_square_check(generator, buckets, probs, nsamples=1000000):
     _, p = ss.chisquare(f_obs=obs_freq, f_exp=expected_freq)
     return p, obs_freq, expected_freq
 
-def verify_generator(generator, buckets, probs, nsamples=1000000, nrepeat=5, success_rate=0.25, alpha=0.05):
+def verify_generator(generator, buckets, probs, nsamples=1000000, nrepeat=5, success_rate=0.15, alpha=0.05):
```

Comment:
why did we change the value from 0.15 to 0.25?

Reply:
Actually it's changed from 0.25 to 0.15, but I have the same question. Why this change?

Reply:
Same question. The comment in this method says "25%" success rate. Why do we change it?

Reply:
@yuxihu @mseth10 @apeforest 
It was changed in this PR: https://github.com/apache/incubator-mxnet/pull/13498/files
Even though It is mentioned in the comments that success rate should be 25%. There isn't any data to support why it should be kep at 25%. It could have been a typo too back then. SO, reverting it to original value seemed more logical.

Reply:
My concern is due to increasing the strinctness of this check for all generators it might caouse them to become flaky. I can also explicitly relax the criteria for only this generator and we can observe the behaviour of others and make changes accordingly. Setting it to a fixed value for all the generators doesn't seem like a good idea since all the random generators are different.

Reply:
It looks like @ChaiBapchya increased it from 0.15 to 0.25 in the PR in order to fix another flaky test. The question now: if you change back to 0.15, will the other test become flaky again?

Reply:
@yuxihu : Actually No. He in fact made the check more strict. I am simply relaxing it. Therefore it shouldn't make another test flaky. But to make our tests better what I am suggesting is that i let it stay at 0.25 as default and explicitly pass 0.15 for exponential generator check. If others don't fail maybe only this one needed adjustment. Do you think this approach is better ?

Reply:
I think its fine to revert it to 0.25 to reduce flakiness. it wont make the other test flaky since this change is decreasing the failure chances.

Reply:
@access2rohit I like your idea of passing 0.15 for exponential generator check.

Reply:
@access2rohit Please make the change as you commented: make 0.25 as default and pass in 0.15 for exponential generator check.

Reply:
@yuxihu @apeforest @mseth10 @anirudh2290 : I have updated successrate to 20% which is less stricter but not too loose a check. Currently running it 10k times and will update the PR once they succeed

Reply:
All tests passed with success_rate=0.20